definitions.yaml

```diff
@@ -55,6 +55,19 @@ definitions:
               default:
                 description: The instance type used in new cluster, if not specified
                 type: string
+          docker_volume_size:
+            description: Size of the docker volume attached to the node. This volume is used to store docker images.
```

Comment:
The line wrapping is enforced here, so these lines are probably too long.

Reply:
there is only a warning about the length. the errors were about trailing whitespace. I've removed it.